docs/api/graphql/getting-started.md

```diff
@@ -17,62 +17,48 @@ We'll reiterate a few important points before proceeding:
-you'd need to a) log into DataHub, b) extract the PLAY_SESSION cookie that is set on login, and c) provide this Cookie in your HTTP headers when
-calling the endpoint.
-   
+#### Authentication + Authorization
 
-### Querying the Endpoint
+In general, you'll need to provide an [Access Token](../../authentication/personal-access-tokens.md) when querying the GraphQL by
+providing an `Authorization` header containing a `Bearer` token. The header should take the following format:
 
-There are a few options when it comes to querying the GraphQL endpoint. The recommendation on which to use varies by use case.
+```bash
+Authorization: Bearer <access-token>
+```
 
-**Testing**: [Postman](https://learning.postman.com/docs/sending-requests/supported-api-frameworks/graphql/), GraphQL Explorer (described below), CURL
+Authorization for actions exposed by the GraphQL endpoint will be performed based  on the actor making the request.
+For Personal Access Tokens, the token will carry the user's privileges. 
 
-**Production**: GraphQL [Client SDK](https://graphql.org/code/) for the language of your choice, or a basic HTTP client.
-   
-> Important: The DataHub GraphQL endpoint only supports POST requests at this time. It does not support GET requests. If this is something
-> you need, let us know on Slack!
+> Notice: The DataHub GraphQL endpoint only supports POST requests at this time.
 
 ### On the Horizon
 
-- **Service Access Tokens**: In the near future, the DataHub team intends to introduce service users, which will provide a way to generate and use API access
+- **Service Tokens**: In the near future, the DataHub team intends to introduce service users, which will provide a way to generate and use API access
 tokens when querying both the Frontend Proxy Server and the Metadata Service. If you're interested in contributing, please [reach out on our Slack](https://datahubspace.slack.com/join/shared_invite/zt-nx7i0dj7-I3IJYC551vpnvvjIaNRRGw#/shared-invite/email).
 - **DataHub Client SDKs**: Libraries wrapping the DataHub GraphQL API on a per-language basis (based on community demand). 
 
 ## GraphQL Explorer 
 
-DataHub provides a browser-based GraphQL Explorer Tool ([GraphiQL](https://github.com/graphql/graphiql)) for live interaction with the GraphQL API. Today, this tool is available for use in multiple places (like the GraphQL endpoint itself):
-
-1. **Metadata Service**: `http://<metadata-service-address>/api/graphiql`. For local deployments, `http://localhost:8080/api/graphiql`.
-2. **Frontend Proxy**: `http://<frontend-service-address>/api/graphiql`. For local deployments, `http://localhost:9002/api/graphiql`.
-
+DataHub provides a browser-based GraphQL Explorer Tool ([GraphiQL](https://github.com/graphql/graphiql)) for live interaction with the GraphQL API. This tool is available at the path `/api/v2/graphiql` (e.g. `https://my-company.datahub.com/api/v2/graphql`)
```

Comment:
oh there's a `v2` now? also I think you need to end the example link with `graphiql` not just `graphql`

Reply:
Yes good catch! Actually v2 is an old thing we can do the normal one